packages/modules/desktop-viewer-react/package.json

```diff
@@ -44,6 +44,7 @@
     "@itwin/core-telemetry": "^3.2.0",
     "@itwin/ecschema-metadata": "^3.2.0",
     "@itwin/electron-authorization": "^0.8.3",
+    "@itwin/imodels-client-management": "^1.0.1",
```

Comment:
can we make this a direct dep in the desktop viewer?

Reply:
Moved it to `dependencies`. 

Ideally, I think we would want this in peer (like other _iTwin_ packages), but in this case it would be a breaking change.